larpandoracontent/LArObjects/LArMCParticle.h

```diff
@@ -27,7 +27,9 @@ namespace lar_content
 class LArMCParticleParameters : public object_creation::MCParticle::Parameters
 {
 public:
-    pandora::InputInt   m_nuanceCode;               ///< The nuance code
+    pandora::InputInt                          m_nuanceCode;        ///< The nuance code
+    std::vector<pandora::InputCartesianVector> m_mcStepPositions;   ///< The positions of the geant4 steps
+    std::vector<pandora::InputCartesianVector> m_mcStepMomentas;    ///< The momenta of the geant4 steps
```

Comment:
Steve's original code uses these `InputCartesianVector`s, which I've left as is, but wasn't sure if/what benefit they had over the `CartesianVector` in this case.

Reply:
This is largely about providing a wrapper that can check the initialisation status for values that come from external sources.